pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -229,7 +229,12 @@ func (m *kubeGenericRuntimeManager) GetPods(all bool) ([]*kubecontainer.Pod, err
 	if err != nil {
 		return nil, err
 	}
-	for _, s := range sandboxes {
+	for i := range sandboxes {
```

Comment:
Items in sandboxes are pointers, it makes a copy of those items to avoid changing them in following codes.